_data/projects/GoGE.yml

```diff
@@ -0,0 +1,15 @@
+name: eBooker
+  desc: GoGE is a two-dimensional, console-based game engine that was programmed in a language called Go.
```

Comment:
You need to remove the whitespace from this line down to line 4 to make the YAML file parseable again